build_runner/lib/src/generate/build_impl.dart

```diff
@@ -443,6 +515,10 @@ class BuildImpl {
         assert(inputNode != null, 'Asset Graph is missing $input');
         inputNode.outputs.add(output);
       }
+
+      // And finally compute the combined digest for all inputs.
```

Comment:
some of these comments are "what" with no "why" - should we be refactoring to make the code easier to understand?

Reply:
broke out some parts of this into appropriately named functions so its easier to follow